www/core/test/list_test.xml

```diff
@@ -1,7 +1,7 @@
 <extensionset name="Joomla Core Test Updateserver" description="The Joomla Core Updateserver for Tests of Alpha, Beta und RC Releases">
```

Comment:
I think this was not about good vs. bad english in this case but in general you are correct 😄